packages/sdk/src/utils/constants.ts

```diff
@@ -16,3 +14,17 @@ export const editionInitFlags = {
   MINT_RANDOMNESS_ENABLED: 2, // 1 << 1
   OPERATOR_FILTERING_ENABLED: 4, // 1 << 2
 } as const
+
+export const UINT32_MAX = 4294967295
+
+export const NULL_ADDRESS = '0x0000000000000000000000000000000000000000'
+
+export const NON_NULL_ADDRESS = '0x0000000000000000000000000000000000000001'
+
+export const NULL_BYTES32 = '0x0000000000000000000000000000000000000000000000000000000000000000'
+
+export const SOUND_FEE = 0
+
+export const ONE_HOUR = 3600
+
+export const PRICE = 420420420
```

Comment:
these are test specific and should just be in the test file as they were before
don't want someone to think sound_fee or price are these values